.github/renovate.json

```diff
@@ -5,14 +5,8 @@
   "semanticCommits": "enabled",
   "reviewers": ["@Jolg42", "@millsp", "@aqrln", "@SevInf", "@jkomyno"],
   "rebaseWhen": "conflicted",
-  "ignoreDeps": [
-    "sqlite3",
-    "@prisma/engines",
-    "@prisma/engines-version",
-    "@prisma/fetch-engine",
-    "@prisma/get-platform",
-    "@prisma/prisma-fmt-wasm"
```

Comment:
Why are the others not needed any more? Because we moved them to the monorepo?

Reply:
Yes, only `engines-version` stayed out when we did that move.